public/sass/course.scss

```diff
@@ -8,7 +8,8 @@ $white: #F5F5F5;
     height: 80px;
     margin-left: auto;
     margin-right: auto;
-    border: 1px solid lightgrey;
+    //border: 1px solid lightgrey;
```

Comment:
You can just delete this. Not that I really mind the commenting out, but since we have it in source control, even if this WAS a critical thing, we could always just go back and find it